src/common/ipc/update-available/index.ts

```diff
@@ -0,0 +1,48 @@
+import { UpdateInfo } from "electron-updater";
```

Comment:
I'm not sure why we need to put these modules under `ipc/`? If we go with this route every module that uses `ipc` should end up here (which sounds wrong).

Reply:
So you would prefer having these checking function under where? I put them here because these IPC types should be "public" to both `renderer` and `main` (as the receiver and sender) and I figured that the verification functions should be beside the type definitions.

Reply:
Ok, makes sense if this is just shared functionality.